scripts/download/taplo.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_dir=$(readlink -f $(dirname $0))
+
+. $script_dir/lib.sh
+
+version=0.7.0
```

Comment:
There appears to be a new version of this tool :)

```suggestion
version=0.8.1
```

Reply:
Good catch. I somehow missed this upgrade, even though thought about it.